pkg/network/tracer/connection/failure/matching.go

```diff
@@ -49,6 +46,12 @@ type FailedConnStats struct {
 	Expiry         int64
 }
 
+func (t FailedConnStats) reset() {
+	for k := range t.CountByErrCode {
```

Comment:
Note: this will not cause the memory usage of the map to be reclaimed. https://github.com/golang/go/issues/20135